### PR TITLE
Backend hist

### PR DESCRIPTION
**Pull request by rmenegaux**
Same as #429, but keeping the old `Hist` and naming the backend Hist `Bins`

### CODE REVIEW

File: bqplot/marks.py [hunk 1]
```diff
@@ -931,6 +933,101 @@ def _validate_orientation(self, proposal):
...
+    # Mark decoration
+    icon = 'fa-signal'
+    name = 'Backend Histogram'
+
+    # Scaled Attributes
+    sample = Array([]).tag(
+        sync=True, display_name='Sample', rtype='Number',
+        atype='bqplot.Axis', **array_serialization
+        ).valid(array_squeeze, array_dimension_bounds(1, 1))
+
```
Comment (maartenbreddels) [inline bqplot/marks.py]: sync=False right, I guess this should not be send to the front-end?

Comment (rmenegaux) [reply bqplot/marks.py]: Yes good catch

**Comment by rmenegaux** [comment]
@maartenbreddels updated according to your comments, does it lgty?

**Comment by maartenbreddels** [comment]
It has my blessings 👍

**Pull request merged**